db/migrate/20170421153056_add_email_count_to_signatures.rb

```diff
@@ -0,0 +1,5 @@
+class AddEmailCountToSignatures < ActiveRecord::Migration
+  def change
+    add_column :signatures, :email_count, :integer
```

Comment:
There may be performance issues with doing this though, not sure if adding default 0 to this column might take ages given how big this table is.